web/docs/Backend Starters/Decoupled WordPress/updating-dependencies.md

```diff
@@ -0,0 +1,17 @@
+# Updating Drupal Dependencies
+
+Dependencies can be updated following two ways:
+
+## 1. Update dependencies manually using composer
+
+The Wordpress(Bedrock) project uses composer to manage dependencies. The dependencies can be updated using the following composer command:
+
+- Update Contributed plugin/theme: `composer update pantheon-systems/plugin-name --with-dependencies`
```

Comment:
This won't always be under the pantheon-systems namespace, especially if additional plugins have been added. I'd maybe make the example here <namespace>/<packagename> and then maybe follow with a real world example as shown here: https://docs.roots.io/bedrock/master/composer/#plugins

`composer update wpackagist-plugin/akismet --with-dependencies`

I'd encourage you to test this, but my understanding is that the WordPress core dependency is currently part of pantheon-systems/wordpress-integrations so I'd expect:

`composer update pantheon-systems/wordpress-integrations --with-dependencies` to result in the update.

Reply:
I ran the above said command it shows all updated. But I was not stratified so tried checking for the repo on Github. Why is so that I'm not able to find any repo named `wordpress-integrations` under the https://github.com/pantheon-systems ?

Reply:
Also I wanted to add a section to update the WordPress core but given the WordPress core is under `pantheon-systems` namespace. I'm not sure what should be done here.